docs(schema-review): add features/schema-review docs (#234)
bytebase/bytebase.com: content/docs/en/use-bytebase/schema-review/create-schema-review-policy.md

---
title: Create Schema Review Policy
---

There are two ways to create the `Schema Review Policy`.

One is from `Schema Review Policy` dashboard in `Settings`.

![schema-review-create-from-policy-dashboard](/static/docs-assets/schema-review-create-from-policy-dashboard.gif)

The other is from `Environment` dashboard.

![schema-review-create-from-environment-dashboard](/static/docs-assets/schema-review-create-from-environment-dashboard.gif)

> The `Schema Review Policy` is a 3-step creation.

Reviewer: ```suggestion
Creating the `Schema Review Policy` is a 3-step process.
```

## Step 1

Specify the basic information, including display name for schema review policy, selected environment and schema review policy template.

<hint-block type="info">

> Note that only **ONE** policy can be created per `Environment`.

Reviewer: ```suggestion
Note that only **ONE** policy can be attached per `Environment`.
```

</hint-block>

## Step 2

Specify each `Schema Review Rule`, including error level and rule-specific fields. 

### Change rule level

You can choose one of `Error`, `Warning` and `Disabled`. 

`Error` will block the automatic run of `Issue`, while `Warning` will not.

When the rule is `Disabled`, it will not take effect.

![schema-review-change-rule-level](/static/docs-assets/schema-review-change-rule-level.gif)

### Change rule configuration

> There are currently three types of rules that need to be configuration.

Reviewer: ```suggestion
There are currently three types of rules that need to be configured.
```
